internal/cmd/create.go

```diff
@@ -46,6 +46,10 @@ func (o *createOptions) Validate(args []string) error {
 		return errors.New("status can only be specified once")
 	}
 
+	if o.ImpactStatement != "" && o.Status != string(vex.StatusNotAffected) {
+		return errors.New("--impact-statement can only be set when using status is not_affected" + o.Status)
```

Comment:
Here's the error message I get with this (notice the end of the message):

```console
$ go run . create --product 'foo' --vuln 'CVE-1999-99999' --status 'fixed' --impact-statement 'oopsies'
Error: --impact-statement can only be set when using status is not_affectedfixed
```

How about something like this?

```suggestion
		return fmt.Errorf("--impact-statement can be set only when status is \"not_affected\" (status was %q)", o.Status)
```

Which changes the error message to:

```console
Error: --impact-statement can be set only when status is "not_affected" (status was "fixed")
```

Reply:
Weird, I don't know why I wrote that, I've changed the error to your suggestion above.